JenkinsConsoleUtility/jcuSrc/Commands/CalendarAlertCommand.cs

```diff
@@ -49,9 +49,9 @@ public int Execute(Dictionary<string, string> argsLc, Dictionary<string, string>
                 var alertTime = expireTime - TimeSpan.FromDays(alertDays);
 
                 if (today >= expireTime)
-                    expiredAlerts.Add($"Expired alert: {eachAlert.name} on: {eachAlert.date}");
+                    expiredAlerts.Add("Expired alert: " + eachAlert.name + " on: " + eachAlert.date);
```

Comment:
Aww, but that's such a cool "new" feature :( But yeah, makes back-compat hard

Reply:
/agree